src/main/java/mezz/jei/gui/GuiScreenHelper.java

```diff
@@ -208,13 +208,7 @@ public IGuiClickableArea getGuiClickableArea(ContainerScreen guiContainer, doubl
 	@Nullable
 	private static <T extends ContainerScreen> IGuiClickableArea getGuiClickableArea(Class<? extends T> handlerClass, IGuiContainerHandler<T> handler, ContainerScreen containerScreen, double mouseX, double mouseY) {
 		if (handlerClass.isInstance(containerScreen)) {
-			T castContainer = handlerClass.cast(containerScreen);
-			Collection<IGuiClickableArea> guiClickableAreas = handler.getGuiClickableAreas(castContainer);
-			for (IGuiClickableArea guiClickableArea : guiClickableAreas) {
-				if (MathUtil.contains(guiClickableArea.getArea(), mouseX, mouseY)) {
-					return guiClickableArea;
-				}
-			}
+			return handler.getGuiClickableArea(handlerClass.cast(containerScreen), mouseX, mouseY);
```

Comment:
please keep one function call per line, so this should be on its own line:
```java
T castContainer = handlerClass.cast(containerScreen);
```
I use this style for clarity and to help narrow down errors in stack traces.